components/automate-ui/src/app/pages/team-management/team-management.component.ts

```diff
@@ -169,10 +162,12 @@ export class TeamManagementComponent implements OnInit, OnDestroy {
   }
 
   public openCreateModal(): void {
-    if (this.isMajorV1) {
-      this.createV1TeamModalVisible = true;
-    } else {
+    let isV2 = true;
+    this.isIAMv2$.subscribe(latest => isV2 = latest);
```

Comment:
This will leak subscriptions a lot. These lines need to be part of the initialization for the file (`ngOnInit`).
Also, because you need the value _both_ in the code-behind and in the template, you can make the template cleaner. Instead of multiple uses of `isIAMv2$ | async` you can just use `isV2`.

Reply:
Oh, great-- I was actually trying to understand this a little more. Wasn't clear on when I could get away with that vs the async call.